src/main/java/uk/gov/hmcts/juror/api/moj/service/ExcusalResponseServiceImpl.java

```diff
@@ -89,7 +89,14 @@ public void respondToExcusalRequest(BureauJwtPayload payload, ExcusalDecisionDto
 
         if (excusalDecisionDto.getReplyMethod() != null) {
             if (excusalDecisionDto.getReplyMethod().equals(ReplyMethod.PAPER)) {
-                setPaperResponseProcessingStatusToClosed(payload, jurorNumber);
+                if (null == jurorPaperResponseRepository.findByJurorNumber(jurorNumber)) {
+                    // There are scenarios where a juror may not have a paper response when excused from the juror
+                    // record
+                    log.info(String.format("No Paper response found for Juror %s when processing excusal request",
+                        jurorNumber));
+                } else {
+                    setPaperResponseProcessingStatusToClosed(payload, jurorNumber);
+                }
             } else {
                 setDigitalResponseProcessingStatusToClosed(payload, jurorNumber);
```

Comment:
Is it possible the digital response doesn't exist? Can this method be reached to process an excusal from the juror record? Potentially if the juror was deferred for a year then excused, housekeeping would have cleared up the digital response

Reply:
If there isnt a response in the db, the default response would be paper, so gets handled in the updated code.